assets/js/vue/utils/template-switch.js

```diff
@@ -14,50 +14,37 @@ const templateVariables = {
     productTypeDesc: 'Product Type',
     tags: 'Tags',
   },
+  GRIIDC: {
+  },
   GRP: {
-    name: 'GRP',
-    status: 'Dataset Status',
     fundingCycle: 'Grant Awards',
     researchGroup: 'Projects',
-    fundingOrg: 'Funding Organizations',
     showFundingCycles: true,
     showProjectDirector: true,
     showFundingOrgFacet: false,
     researchAwardOption: 'Grant Award',
     cardHeadingText: 'View Project Overview By',
-    digitalTypeDesc: 'Resource Type',
-    productTypeDesc: 'Product Type',
-    tags: 'Tags',
   },
   HRI: {
-    name: 'HRI',
-    status: 'Dataset Status',
-    fundingCycle: 'Funding Cycles',
-    researchGroup: 'Research Groups',
-    fundingOrg: 'Funding Organizations',
     showFundingCycles: false,
     showProjectDirector: false,
     showFundingOrgFacet: false,
-    researchAwardOption: 'Research Award',
-    digitalTypeDesc: 'Resource Type',
-    productTypeDesc: 'Product Type',
-    tags: 'Tags',
   },
 };
 
 export default {
   getProperty: (name) => {
     if (
-    // eslint-disable-next-line no-undef
-      PELAGOS_TEMPLATE_PROPS !== undefined
-      // eslint-disable-next-line no-undef
-            && PELAGOS_TEMPLATE_PROPS.BaseTemplateName !== undefined
-      // eslint-disable-next-line no-undef
-            && templateVariables[PELAGOS_TEMPLATE_PROPS.BaseTemplateName] !== undefined
+      global.baseTemplateName !== undefined
+            && global.baseTemplateName !== undefined
```

Comment:
repeated from above...as we found earlier

Reply:
You didn't refresh, that's old.